discoverer/src/test/java/eu/seaclouds/platform/discoverer/core/OfferingTest.java

```diff
@@ -1,10 +1,5 @@
 package eu.seaclouds.platform.discoverer.core;
 
-import alien4cloud.model.topology.NodeTemplate;
-import org.testng.Assert;
-import org.testng.annotations.Test;
-
-
 public class OfferingTest {
```

Comment:
Since alien4cloud was removed, this test became totally useless. I will remove this file.